apps/custom-config/vt-word.js

```diff
@@ -70,16 +72,28 @@ class VtWord extends BaseConfig {
       if (realQueryObj.action === SWITCH_VT_UPDATE) {
         logger.info(`turen update first load ${realQueryObj.txt}`)
         this.activity.turen.deleteVtWord(realQueryObj.oldTxt)
-        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py)
+        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py, realQueryObj.margin_index, realQueryObj.cloud_confirm)
       } else if (realQueryObj.action === SWITCH_VT_ADD) {
-        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py)
+        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py, realQueryObj.margin_index, realQueryObj.cloud_confirm)
       } else if (realQueryObj.action === SWITCH_VT_DELETE) {
         this.activity.turen.deleteVtWord(realQueryObj.txt)
       }
       this.activity.exit()
     }
   }
 
+  assertVtWordValueValid (config) {
+    if (config) {
+      if (!config.margin_index) {
```

Comment:
if `config.margin_index` is `0`, `!config.margin_index` return `true`

Reply:
updated, plz check again